modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/GroupUpdateRequest.java

```diff
@@ -149,10 +146,17 @@ public CompletableFuture<Void> handle(DisasterRecoveryManager disasterRecoveryMa
         Catalog catalog = disasterRecoveryManager.catalogManager.catalog(catalogVersion);
 
         CatalogZoneDescriptor zoneDescriptor = catalog.zone(zoneId);
-        CatalogTableDescriptor tableDescriptor = catalog.table(tableId);
+
+        Set<Integer> allZonePartitionsToReset = new HashSet<>();
+        partitionIds.values().forEach(allZonePartitionsToReset::addAll);
 
         CompletableFuture<Map<TablePartitionId, LocalPartitionStateMessageByNode>> localStates
-                = disasterRecoveryManager.localPartitionStatesInternal(Set.of(zoneDescriptor.name()), emptySet(), partitionIds, catalog);
+                = disasterRecoveryManager.localPartitionStatesInternal(
```

Comment:
That is very interesting way of formatting...
You never cease to amaze me :)

Reply:
Fixed